frontend/src/scenes/experiments/MetricsView/shared/utils.ts

```diff
@@ -182,9 +182,9 @@ export function getIntervalLabel(result: ExperimentVariantResult): string {
 
 export interface PercentageChangeResult {
     text: string
-    isSignificant: boolean
-    isPositive: boolean | null
-    pointEstimate: number | null
+    isSignificant: boolean | undefined
+    isPositive: boolean | undefined
+    pointEstimate: number | undefined
```

Comment:
This is an anti-pattern; instead, we should use `isSignificant?: boolean`

Reply:
Fixed in 6ca17fd87d6ab9525e7d96089f6c0923ccd8acaa